testing/fixtures.py

```diff
@@ -38,7 +39,10 @@ def copy_tree_to_path(src_dir, dest_dir):
 
 def git_dir(tempdir_factory):
     path = tempdir_factory.get()
-    cmd_output('git', 'init', path)
+    try:
+        cmd_output('git', 'init', '--initial-branch=master', path)
+    except CalledProcessError:  # pragma: no cover (pre 2.28 git)
+        cmd_output('git', 'init', path)
```

Comment:
without needing special version-specific coverage:

```bash
git -c init.defaultBranch=master init path
```

Reply:
OK, didn't realize that worked. I'll fix that when I'm back from running a few errands. Thanks!